tests/conftest.py

```diff
@@ -41,6 +41,8 @@ def event_loop(request):
     loop.close()
 
     gc.collect()
+    # for pypy
```

Comment:
best practices recommend placing inline comments in the same line they refer to: this way you ensure that it won't get decoupled by some accidental changes in future.